ts/Core/Series/SeriesType.d.ts

```diff
@@ -41,7 +41,10 @@ export type SeriesTypeOptions = SeriesType['options'];
  * All possible series plotOptions.
  */
 export type SeriesTypePlotOptions = {
-    [K in keyof SeriesTypeRegistry]?: Omit<SeriesTypeRegistry[K]['prototype']['options'], NonPlotOptions>;
+    [K in keyof SeriesTypeRegistry]?: Omit<
+    SeriesTypeRegistry[K]['prototype']['options'],
+    NonPlotOptions
```

Comment:
Why not 1 more indent for the lines from between `<` and `>` ?

Reply:
I agree, but the linter wouldn't allow it... Looks like a bug to me.

Reply:
BTW, sorry for tagging you in the weekend. Didn't mean to, I must have accidentally clicked your name :)

Reply:
(😃 Don't worry, I wasn't tagged. It's still a weekday for me (but not for long) - just going through issues and this poped-up.)